hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/Action.java

```diff
@@ -168,7 +168,13 @@ default boolean isForced() {
         /**
          * Cancellation has been rejected by the controller.
          */
-        CANCEL_REJECTED;
+        CANCEL_REJECTED,
+
+        /**
+         * Action has been downloaded by the target and waiting for update to
+         * start.
+         */
+        DOWNLOADED;
```

Comment:
You need to map the enum in the `JpaActionStatus` and `JpaAction` in the `@ObjectTypeConverter`. Otherwise you retrieve an exception when using it to give a `downloaded` type feedback. 
`No conversion value provided for the attribute [DOWNLOADED]`
```
@ObjectTypeConverter(name = "status", objectType = Action.Status.class, dataType = Integer.class, conversionValues = {
            @ConversionValue(objectValue = "FINISHED", dataValue = "0"),
            @ConversionValue(objectValue = "ERROR", dataValue = "1"),
            @ConversionValue(objectValue = "WARNING", dataValue = "2"),
            @ConversionValue(objectValue = "RUNNING", dataValue = "3"),
            @ConversionValue(objectValue = "CANCELED", dataValue = "4"),
            @ConversionValue(objectValue = "CANCELING", dataValue = "5"),
            @ConversionValue(objectValue = "RETRIEVED", dataValue = "6"),
            @ConversionValue(objectValue = "DOWNLOAD", dataValue = "7"),
            @ConversionValue(objectValue = "SCHEDULED", dataValue = "8"),
            @ConversionValue(objectValue = "CANCEL_REJECTED", dataValue = "9"),
            @ConversionValue(objectValue = "DOWNLOADED", dataValue = "10") })
```